### ISSUE TITLE
Allow seeding one or more possible solutions to the GA

### ISSUE THREAD

**Issue by Gabriel-p** [asteca/ASteCA#265]
The user can seed possible solutions by adding them to the initial random population generated by the GA. One or more possible solutions can be added in the `random_population` function of the GA:

```
p_lst_r = random_population(param_values, n_pop)
```

To input these solutions, these lines can be added to the `params_input.dat` file:

```
# Seed solutions
# z
SZ   [0.001, 0.025]
# log(age)
SA    [7.5, 9.8]
# E(B-V)
SE   [0.02, 0.15]
# dm
SD    [10.1, 13.]
# Mass
SM    [5000., 10000.]
# bf
SB    [0.1, 0.8]
```

The results should be stored the same way the _top tiers_ solutions are stored. This is, plot the 10 first (if more than 10 solutions passed) solutions with the smaller likelihoods, and store the resulting parameter values and likelihoods for all solutions passed in a .dat file.

This method is mentioned in [Genetic Algorithms for the Travelling Salesman Problem](http://link.springer.com/article/10.1023/A%3A1006529012972), Sect 5 (Zotero).

**Comment by Gabriel-p** [comment]
The GA is being superseded by the Bayesian method, so this is no longer a priority.

**Comment by Gabriel-p** [comment]
Closed by #472
